unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/GotoPanelHUD/GotoPanelFeature.cs

```diff
@@ -0,0 +1,16 @@
+public class GotoPanelFeature : IPlugin
```

Comment:
```suggestion
public class GotoPanelPlugin : IPlugin
```

Team agreed to maintain a `WhateverNamePlugin` convention for plugins.

Reply:
done